worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitCommandSender.java

```diff
@@ -76,21 +74,21 @@ public void printRaw(String msg) {
     @Override
     public void print(String msg) {
         for (String part : msg.split("\n")) {
-            sender.sendMessage("\u00A7d" + part);
+            sender.sendMessage("§d" + part);
```

Comment:
Is it fine to have these in the code now? It never used to be years back

Reply:
I have made it safe now, enforcing UTF-8 source-reading in all cases.